sonatype-ci-release/src/main/scala/org/typelevel/sbt/TypelevelSonatypeCiReleasePlugin.scala

```diff
@@ -44,6 +44,12 @@ object TypelevelSonatypeCiReleasePlugin extends AutoPlugin {
   override def globalSettings =
     Seq(tlCiReleaseTags := true, tlCiReleaseBranches := Seq())
 
+  private def renderSummaryTable(results: Map[String, String]): String =
+    results
+      .toList
+      .map { case (k, v) => s"| ${k} | ${v} |" }
+      .mkString(s"| Build Results | |\n| -: | :- |\n", "\n", "")
```

Comment:
Since it's not specific to the build step?

```suggestion
      .mkString(s"| Workflow Results | |\n| -: | :- |\n", "\n", "")
```

Reply:
It's hardcoded in the release step though, maybe `Release Workflow Results` is better? I'm wondering whether it will make sense to add this feature in a more agnostic way in to the `ghactions` plugin.

Reply:
> I'm wondering whether it will make sense to add this feature in a more agnostic way in to the `ghactions` plugin. wdyt?

Yes me too. Hmm 😂